tests/Unit/Utils/DistanceUtilsTest.php

```diff
@@ -0,0 +1,37 @@
+<?php
+
+namespace MyParcelCom\ApiSdk\Tests\Unit\Utils;
+
+use MyParcelCom\ApiSdk\Exceptions\ConversionException;
+use MyParcelCom\ApiSdk\Utils\DistanceUtils;
+use PHPUnit\Framework\TestCase;
+
+class DistanceUtilsTest extends TestCase
+{
+    /** @test */
+    public function testDistanceConversion()
+    {
+        $initialDistance = 100;
+        $expectedDistance = $initialDistance * 1000 / 1609.344;
+
+        $convertedDistance = DistanceUtils::convertDistance($initialDistance, 'kilometers', 'miles');
+        $revertedDistance = DistanceUtils::convertDistance($convertedDistance, 'miles', 'kilometers');
+
+        $this->assertEquals($expectedDistance, $convertedDistance);
```

Comment:
I think it would be better to hardcode the expected outcome of these tests, so there are no other factors that influence this test, other than the code itself